packages/core/src/types.ts

```diff
@@ -4,7 +4,11 @@ import type { ConnectionRecord, DidCommService } from './modules/connections'
 import type { AutoAcceptCredential } from './modules/credentials/CredentialAutoAcceptType'
 import type { AutoAcceptProof } from './modules/proofs'
 import type { MediatorPickupStrategy } from './modules/routing'
-import type { Verkey, WalletConfig, WalletCredentials } from 'indy-sdk'
+
+export interface WalletConfig {
+  walletId: string
+  walletKey: string
+}
```

Comment:
Will we put here also other `config` and `credential` properties eventually? 

I think we can get rid of `wallet` prefix:

```suggestion
export interface WalletConfig {
  id: string
  key: string
}
```

Reply:
> Will we put here also other `config` and `credential` properties eventually?

Yes that was the idea. So whenever we support other storage types / wallet types we can create an extended interface for that specific type